lab-carolina/test/test.js

```diff
@@ -0,0 +1,15 @@
+const read = require('../lib/read.js');
+const assert = require('assert');
+
+describe('testing module asyncly', function(){
+  describe('read()', function(){
+    it('should return .txt files in order', function(done){
+      read(function(text){
+        assert.equal(text[0], '7374756666207374');
```

Comment:
You should probably separate these assertions into their own `it` block. It's typically best practice to only test one small piece of your code at any given time.